_script/cibuild-docker

```diff
@@ -38,11 +38,20 @@ function arguments_mgmt() {
 }
 
 function travis_deploy_flags() {
-    exec_env_vars=""
+    docker_deploy_flags=""
     if $(test "$sim_deploy" = "yes")
     then
-        echo "INFO: deploy flag enabled. We will enable the env variables to simulate a travis merge to master"
-        exec_env_vars='-e TRAVIS_PULL_REQUEST="false" -e TRAVIS_BRANCH="master"'
+        echo "INFO: deploy flag enabled. We will activate the env variables, and mount your docker config/credentials,\
+         to simulate a travis merge to master"
+        local_docker_config_store="${HOME}/.docker"
+        if ! $(test -f "${local_docker_config_store}/config.json")
```

Comment:
why are you not checking if you have permissions (-r)?

Reply:
At this stage, it does not make sense: cibuild-docker is run in the environment the user is using. Then, it runs docker as a different user, and is that user the one that requires read access